### PR TITLE
Astro 4/23 release notes

### PR DESCRIPTION
**Pull request by jwitz**
Resolves https://github.com/astronomer/docs/issues/3650

### CODE REVIEW

File: astro/release-notes.md [hunk 1]
```diff
@@ -25,6 +25,14 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
 
 <!-- ALL LINKS TO INTERNAL DOCS MUST BE COMPLETE URLS INCLUDING HTTPS. Otherwise the links will break in RSS. -->
 
+## April 23, 2024
+
+### Additional improvements
+
+- You can now prmotoe a [development Deployment](https://docs.astronomer.io/astro/deployment-resources#hibernate-a-development-deployment) to a production Deployment by switching off the **Development Mode** toggle in the Deployment's configuration.
+- Workspace Members can now make and use [custom Airflow menu plugins](https://docs.astronomer.io/learn/using-airflow-plugins#appbuilder-menu-items). To give a custom role this permission, you can select `deployment.airflow.customMenu.get` in the role's permissions list. Note that this permission works only on Deployments running Astro Runtime 9 or later.
+- You can now filter the Workspaces and clusters lists in the Astro UI by name.
```
Comment (jwitz) [inline astro/release-notes.md:34]:
```suggestion
- You can now filter the Workspaces and clusters lists in the Astro UI by name.

### Bug fixes

- In order to ensure the reliability of data lineage while Astronomer upgrades the Astro data lineage backend, data lineage is now a private preview feature. To reenable data lineage for an Astro Organization, reach out to your account team.
```

File: astro/release-notes.md [hunk 2]
```diff
@@ -25,6 +25,18 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
 
 <!-- ALL LINKS TO INTERNAL DOCS MUST BE COMPLETE URLS INCLUDING HTTPS. Otherwise the links will break in RSS. -->
 
+## April 23, 2024
+
+### Additional improvements
+
+- You can now prmotoe a [development Deployment](https://docs.astronomer.io/astro/deployment-resources#hibernate-a-development-deployment) to a production Deployment by switching off the **Development Mode** toggle in the Deployment's configuration.
+- Workspace Members can now make and use [custom Airflow menu plugins](https://docs.astronomer.io/learn/using-airflow-plugins#appbuilder-menu-items). To give a custom role this permission, you can select `deployment.airflow.customMenu.get` in the role's permissions list. Note that this permission works only on Deployments running Astro Runtime 9 or later.
```
Comment (jwitz) [inline astro/release-notes.md:33]:
```suggestion
- Workspace Members can now make and use [custom Airflow menu plugins](https://docs.astronomer.io/learn/using-airflow-plugins#appbuilder-menu-items). To give a custom role this permission, you can add `deployment.airflow.customMenu.get` to the role's permissions list. Note that this permission works only on Deployments running Astro Runtime 9 or later. 
```

**Review by lzdanski — APPROVED** [review]

File: astro/release-notes.md [hunk 3]
```diff
@@ -25,6 +25,18 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
 
 <!-- ALL LINKS TO INTERNAL DOCS MUST BE COMPLETE URLS INCLUDING HTTPS. Otherwise the links will break in RSS. -->
 
+## April 23, 2024
+
+### Additional improvements
+
+- You can now prmotoe a [development Deployment](https://docs.astronomer.io/astro/deployment-resources#hibernate-a-development-deployment) to a production Deployment by switching off the **Development Mode** toggle in the Deployment's configuration.
```
Comment (chomipi88) [inline astro/release-notes.md:32]:
```suggestion
- You can now promote a [development Deployment](https://docs.astronomer.io/astro/deployment-resources#hibernate-a-development-deployment) to a production Deployment by switching off the **Development Mode** toggle in the Deployment's configuration.
```

**Comment by chomipi88** [comment]
The release notes are missing the updates mentioned in: https://github.com/astronomer/docs/issues/3650

Relevant issues to the new enhancement:
https://github.com/astronomer/astro/issues/19766
https://github.com/astronomer/astro/issues/19956

File: astro/user-permissions.md [hunk 4]
```diff
@@ -65,6 +65,7 @@ The following table lists the specific permissions that each Workspace role has:
 | View Astro Cloud IDE projects                                                                                          | ✔️                    | ✔️                    | ✔️                      | ✔️                   |
 | View Astro alerts                                                                                                      | ✔️                    | ✔️                    | ✔️                      | ✔️                   |
 | View the **Cluster Activity** tab in the Airflow UI                                                                                                      | ✔️                    | ✔️                    | ✔️                      | ✔️                   |
+| Use custom plugins for the Airflow UI menu                                                                                  | ✔️                    | ✔️                    | ✔️                      | ✔️                   |
```
Comment (chomipi88) [inline astro/user-permissions.md:68]:
```suggestion
| Use custom plugins from the Airflow UI menu                                                                                  | ✔️                    | ✔️                    | ✔️                      | ✔️                   |
```

File: astro/release-notes.md [hunk 5]
```diff
@@ -25,6 +25,18 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
...
+- Workspace Members can now make and use [custom Airflow menu plugins](https://docs.astronomer.io/learn/using-airflow-plugins#appbuilder-menu-items). To give a custom role this permission, you can add `deployment.airflow.customMenu.get` to the role's permissions list. Note that this permission works only on Deployments running Astro Runtime 9 or later.
```
Comment (chomipi88) [inline astro/release-notes.md:33]: Not sure if make is accurate here. Workspace Members can't deploy code to Astro. With the new permission they'll only be able to see/use custom menu plugins.

**Review by chomipi88 — APPROVED** [review]

File: astro/release-notes.md [hunk 6]
```diff
@@ -25,6 +25,23 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
...
+
+### Additional improvements
+
+- The [custom Deployment roles](https://docs.astronomer.io/astro/customize-deployment-roles) feature is now generally available.
+- You can now promote a [development Deployment](https://docs.astronomer.io/astro/deployment-resources#hibernate-a-development-deployment) to a production Deployment by switching off the **Development Mode** toggle in the Deployment's configuration.
+- Workspace Members can now see and use [custom Airflow menu items](https://docs.astronomer.io/learn/using-airflow-plugins#appbuilder-menu-items). To give a custom role this permission, you can add `deployment.airflow.customMenu.get` to the role's permissions list. Note that this permission works only on Deployments running Astro Runtime 9 or later. Note that you might have to modify the code for your menu item plugins for them to work on Astro. See [Appbuilder menu items](https://docs.astronomer.io/learn/using-airflow-plugins#appbuilder-menu-items) for more information.
```
Comment (jwitz) [inline astro/release-notes.md:38]:
```suggestion
- Workspace Members can now see and use [custom Airflow menu items](https://docs.astronomer.io/learn/using-airflow-plugins#appbuilder-menu-items). To give a custom role this permission, you can add `deployment.airflow.customMenu.get` to the role's permissions list. This permission works only on Deployments running Astro Runtime 9 or later. 

    Note that you might have to modify the code for your menu item plugins for them to work on Astro. See [Appbuilder menu items](https://docs.astronomer.io/learn/using-airflow-plugins#appbuilder-menu-items) for more information.
```

File: astro/release-notes.md [hunk 7]
```diff
@@ -25,6 +25,25 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
...
+
+### Additional improvements
+
+- The [custom Deployment roles](https://docs.astronomer.io/astro/customize-deployment-roles) feature is now generally available.
+- You can now promote a [development Deployment](https://docs.astronomer.io/astro/deployment-resources#hibernate-a-development-deployment) to a production Deployment by switching off the **Development Mode** toggle in the Deployment's configuration.
+- Workspace Members can now see and use [custom Airflow menu items](https://docs.astronomer.io/learn/using-airflow-plugins#appbuilder-menu-items). To give a custom role this permission, you can add `deployment.airflow.customMenu.get` to the role's permissions list. This permission works only on Deployments running Astro Runtime 9 or later. 
+
+    Note that you might have to modify the code for your menu item plugins for them to work on Astro. See [Appbuilder menu items](https://docs.astronomer.io/learn/using-airflow-plugins#appbuilder-menu-items) for more information.
```
Comment (jwitz) [inline astro/release-notes.md:40]:
```suggestion
    Note that you might have to modify the code for your menu item plugins to make them work on Astro. See [Appbuilder menu items](https://docs.astronomer.io/learn/using-airflow-plugins#appbuilder-menu-items) for more information.
    
```

**Pull request merged**